Assets/Scripts/EOSPlayerDataStorageManager.cs

```diff
@@ -173,7 +137,7 @@ public bool StartFileDataUpload(string fileName, Action fileCreatedCallback = nu
             {
                 LocalUserId = localUserId,
                 Filename = fileName,
-                ChunkLengthBytes = MAX_CHUNK_SIZE,
+                ChunkLengthBytes = MaxChunkSize,
```

Comment:
Double check team coding standards regarding constants - it might be that this should remain underscore all caps.

Reply:
Changed back to `MAX_CHUNK_SIZE`.